packages/core/src/components/discord-embed/discord-embed.tsx

```diff
@@ -145,7 +145,11 @@ export class DiscordEmbed implements ComponentInterface {
 								<slot></slot>
 							</div>
 							<slot name="fields"></slot>
-							<div class={clsx('discord-embed-media', { 'discord-embed-media-video': Boolean(this.video) })}>{this.renderMedia()}</div>
+							{this.image || this.video ? (
```

Comment:
Does this execute as `(this.image || this.video) ? (...) : null` or as `this.image || (this.video ? (...) : null)`?

If ESLint doesn't complain, I think it might be worth to wrap the OR condition with parens.

Reply:
This executes as the former based on comparison operations.

Prettier does complain however, so I think we are on the safe side with this:
![image](https://user-images.githubusercontent.com/20760160/115794874-1a50e900-a3cf-11eb-878a-60180815e856.png)